shared/tracker/action.desktop.js

```diff
@@ -3,10 +3,25 @@ import React, {PureComponent} from 'react'
 import {Button, Text, Icon} from '../common-adapters'
 import {globalStyles, globalColors, globalMargins, desktopStyles} from '../styles'
 import {normal} from '../constants/tracker'
+import type {SimpleProofState} from '../constants/types/tracker'
```

Comment:
update the naming convention to modern and kill all the .native files which were empty renders anyways